_data/whats-new.yml

```diff
@@ -4,8 +4,30 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Fri Aug 2 12:05:50 2019
+updated: Fri Aug 9 23:29:50 2019
 entries:
+- description: Added information about how to [define a custom Import Entity](https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/custom-import-entity.html).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 7, 2019
+  link: https://github.com/magento/devdocs/pull/5007
+- description: Updated the label descriptions for [DevDocs contributions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs), corrected links, and added a link to the [Beginner's Guide](https://github.com/magento/magento2/wiki/Getting-Started).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: August 5, 2019
+  link: https://github.com/magento/devdocs/pull/4551
+- description: Added a new topic about [enabling database query profiling](https://devdocs.magento.com/guides/v2.3/cloud/project/profile-database-queries.html).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5111
+- description: Added a new task topic for [setting the cache time-to-live (TTL) for
+    static files](https://devdocs.magento.com/guides/v2.3/cloud/project/set-cache.html)
+    on Magento Commerce (Cloud).
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 2, 2019
+  link: https://github.com/magento/devdocs/pull/5109
```

Comment:
The original PR still contains the link to `.md` file:
https://github.com/magento/devdocs/pull/5109

Reply:
`whatsnew` in the PR still contains link to `.md` file

Reply:
![image](https://user-images.githubusercontent.com/6391769/62896150-09a13200-bd16-11e9-8271-69ec16788aaf.png)

Reply:
check link in `whatsnew` please

Reply:
@meker12 the link you changed is out of scope for this pull request.
Target content for this PR follows the `whatsnew` tag.

Reply:
<img width="844" alt="Screen Shot 2019-08-12 at 15 52 29" src="https://user-images.githubusercontent.com/12731225/62897821-98637e00-bd19-11e9-9bcb-2813553617e9.png">

Reply:
I fixed it